examples/rest-xml-json/README_pt.adoc

```diff
@@ -0,0 +1,398 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
+= REST Simples
+
+Definir um serviço REST é muito fácil, simplesmente anúncio a anotação @Path para a
+classe, então definir nos métodos, o método HTTP para usar (@GET, @POST, …).
+
+#O Código
+
+== O serviço REST: @Path, @Produces, @Consumes
+
+Aqui nós vemos um bean que utiliza a opção Concorrência Bean-Managed, bem
+como a anotação @Startup, que faz com que o bean seja instanciado pelo  
+contêiner, enquanto a aplicação inicia. Beans Singleton com 
+@ConcurrencyManagement(BEAN) são responsáveis por seus próprios 
+thread-safety. O bean mostrado é uma simples ``registro'' de propriedade e
+provisiona um lugar onde opções podem ser definidas e recuperadas por todos os beans
+na aplicação.
```

Comment:
the example described here does not reflect the current project code. Because it does not contain the elements described in the text. Here it is necessary to update to reflect the example and also to update the English version.

Reply:
the example does not contain annotations like @Startup, @ConcurrencyManagement(BEAN).

Reply:
The change is applied.